Dockerfile

```diff
@@ -1,45 +1,58 @@
+# using tag debian:buster-slim to automatically receive updates on buster
+# assuming debian:buster-slim is stable enough, so on breaking changes

```

Comment:
```suggestion
# assuming debian:buster-slim is stable enough, so no breaking changes

```

Reply:
probeer hier duidelijk te maken dat we aannemen dat er geen breaking changes zijn in  debian:buster-slim

Reply:
dat staat er ook duidelijk. alleen zat er dan een type in het woordje "no" volgens mij. sorry, die "add a suggestion" knop in github maakt het zo makkelijk.

Reply:
aah, pff mijn brein interpreteerde de diff andersom....